postgres-appliance/configure_spilo.py

```diff
@@ -268,6 +276,9 @@ def get_placeholders(provider):
     placeholders.setdefault('PGDATA', os.path.join(placeholders['PGROOT'], 'pgdata'))
     placeholders.setdefault('PGUSER_STANDBY', 'standby')
     placeholders.setdefault('PGPASSWORD_STANDBY', 'standby')
+    placeholders.setdefault('USE_ADMIN', True)
```

Comment:
We don't really need to pass 'USE_ADMIN' from outside, but we can calculate its value from the presence of 'PGPASSWORD_ADMIN' in environment:
```diff
-    placeholders.setdefault('USE_ADMIN', True)
+    placeholders.setdefault('USE_ADMIN', 'PGPASSWORD_ADMIN' in placeholders)
```

If you wish, it would be still possible to pass it.

Reply:
Nice, I like that as well. So it can be overridden if you set `USE_ADMIN` but if you don't it will default to whether or not `PGPASSWORD_ADMIN` or whatever we define is actually there? Sorry first time using mustache templating.